Update user stories (#41)
AY2425S2-CS2103T-T16-3/tp: docs/DeveloperGuide.md

Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`

| Priority | As a …​  | I want to …​                          | So that I can…​                                                          |
|---------|----------|---------------------------------------|--------------------------------------------------------------------------|

> | `* * *` | user     | add a new applicant contact           | start adding new applicant's details into the application quickly        |

Reviewer: ```suggestion
| `* * *` | user     | add a new applicant's contact details           | start adding new applicant's details into the application quickly        |
```

> | `* * *` | user     | list all applicant's contact          | verify the stored data                                                   |

Reviewer: ```suggestion
| `* * *` | user     | list all applicants' contacts          | verify the stored data                                                   |
```

> | `* * *` | user     | find an applicant's contact           | locate details of persons without having to go through the entire list   |

Reviewer: ```suggestion
| `* *` | user     | find an applicant's contact           | locate details of persons without having to go through the entire list   |
```

| `* * *` | user     | delete applicant's contact            | remove applicants that are no longer appying for a job                   |
| `* * `  | new user | view usage instructions               | refer to instructions when I forget how to use the application           |

> | `* *`   | user     | edit an applicant's contact           | reflect any changes to the applicant's info in a single step             |

Reviewer: ```suggestion
| `* *`   | user     | edit an applicant's contact           | rectify any discrepancies in the applicant's contact details             |
```

> | `* *`   | new user | import my list of applicant's contact | seamlessly migrate data from using other application to this application |

Reviewer: ```suggestion
| `* *`   | new user | import my list of applicant's contact | seamlessly migrate data from one device to another |
```

> | `*`     | new user | play around with sample data          | locate a person easily                                                   |

Reviewer: ```suggestion
| `*`     | new user | play around with sample data          | gain more familiarity with using the app                                                   |
```
